cmd/agent/app/flags.go

```diff
@@ -22,69 +22,84 @@ package app
 
 import (
 	"flag"
-	"fmt"
 	"strings"
+
+	"github.com/spf13/viper"
+)
+
+const (
+	suffixWorkers             = "workers"
+	suffixServerQueueSize     = "server-queue-size"
+	suffixServerMaxPacketSize = "server-max-packet-size"
+	suffixServerHostPort      = "server-host-port"
+	collectorHostPort         = "collector.host-port"
+	httpServerHostPort        = "http-server.host-port"
+	discoveryMinPeers         = "discovery.min-peers"
 )
 
-// Bind binds the agent builder to command line options
-func (b *Builder) Bind(flags *flag.FlagSet) {
+// CreateFlags returns a new FlagSet that defines parameters of the Builder where default
+// flag values are retrieved from the current values of the Builder's fields
+func (b *Builder) CreateFlags() *flag.FlagSet {
```

Comment:
I don't have any :). I thought It's better to go with standard lib if there is not need for cusom behaviour provided by cobra/pflags